roles/lnls-ans-role-desktop-apps/vars/Debian-stretch.yml

```diff
@@ -13,7 +13,7 @@ __desktop_packages:
   - "xfce4-goodies={{ pkg_version_xfce4_goodies }}"
   - "wmctrl={{ pkg_version_wmctrl }}"
   - "cups={{ pkg_version_cups }}"
-  - "ca-certificates={{ pkg_version_ca_certificates }}"
+  - ca-certificates
```

Comment:
why should we use a different version-control approach with this package?

Reply:
This package gets updated every now and thenm and the older versions will cease to exist. For instance, I couldn't install the older 20161130* version on a fresh debian9 system. I think this would simplify maintenance for this. What do you think?

Reply:
i see. if it does get updates that often we could use a particular version-control policy for it.

Reply:
ok, nice! I will keep it like this, then.